react-debugging.html

```diff
@@ -63,11 +63,12 @@
           // No need to change anything ABOVE this line ^
 
           return(
+            <>
```

Comment:
Yes, following the prompts from the console definitely gets you to the solution quicker. Great job!